internal/gengapic/testdata/empty_example.want

```diff
@@ -351,6 +353,8 @@ func ExampleClient_GetOperation() {
 }
 
 func ExampleClient_DeleteOperation() {
+	// import longrunningpb "google.golang.org/genproto/googleapis/longrunning"
```

Comment:
Should the `// import mypackagepb "mypackage"` also be included in addition?

Reply:
In this example, no, because this is a "mix in" method - an RPC form the Operations service we are surfacing on the client b.c the API exposes it.